src/ctap/timed_permission.rs

```diff
@@ -118,71 +116,84 @@ impl U2fUserPresenceState {
 #[cfg(test)]
 mod test {
     use super::*;
-    use core::isize;
-
-    const CLOCK_FREQUENCY_HZ: usize = 32768;
-    const ZERO: ClockValue = ClockValue::new(0, CLOCK_FREQUENCY_HZ);
-    const BIG_POSITIVE: ClockValue = ClockValue::new(isize::MAX / 1000 - 1, CLOCK_FREQUENCY_HZ);
-    const NEGATIVE: ClockValue = ClockValue::new(-1, CLOCK_FREQUENCY_HZ);
-    const SMALL_NEGATIVE: ClockValue = ClockValue::new(isize::MIN / 1000 + 1, CLOCK_FREQUENCY_HZ);
-    const REQUEST_DURATION: Duration<isize> = Duration::from_ms(1000);
-    const PRESENCE_DURATION: Duration<isize> = Duration::from_ms(1000);
-
-    fn grant_up_when_needed(start_time: ClockValue) {
-        let mut u2f_state = U2fUserPresenceState::new(REQUEST_DURATION, PRESENCE_DURATION);
+
+    fn zero() -> CtapInstant {
+        CtapInstant::new(0)
+    }
+
+    fn big_positive() -> CtapInstant {
+        CtapInstant::new(u64::MAX / 1000 - 1)
+    }
+
+    fn request_duration() -> Milliseconds<u64> {
+        Milliseconds::new(1000_u64)
+    }
+
+    fn presence_duration() -> Milliseconds<u64> {
+        Milliseconds::new(1000_u64)
+    }
+
+    fn epsilon() -> Milliseconds<u64> {
+        Milliseconds::new(1_u64)
```

Comment:
Nit: Those `_u64` could be removed. They are already part of the return type.

Reply:
Done.